DFKPS/DFKPS-IOC-01App/Db/DFKPS_8800_status.db

```diff
@@ -344,8 +344,8 @@ record(ao, "$(P)SIM:RAW"){
     field(DTYP, "Soft Channel")
     field(EGU, "A")
 }
-alias("$(P)SIM:CURR", "$(P)SIM:RAW:SP")
-alias("$(P)SIM:CURR", "$(P)SIM:RAW")
+alias("$(P)SIM:RAW", "$(P)SIM:RAW:SP")
+alias("$(P)SIM:RAW", "$(P)SIM:CURR")
```

Comment:
That would be it, the SIM:CURR isn't actually appropriate any more so I shall delete it.